### PR TITLE
Bump services to 0.40.1

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
- Bumps network node version for local and previewnet configs to `0.40.1`
- Bumps mirror node version for local and previewnet configs to `0.84.0`
- Bumps relay version to `0.27.1`
- Fixes a breaking change from services by updating network `config.txt` format.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by Nana-EC** [comment]
@Ivo-Yankov  Can bump them to the following instead

> RELAY_IMAGE_TAG: 0.27.1
> NETWORK_NODE_IMAGE_TAG: 0.40.1
> HAVEGED_IMAGE_TAG: 0.40.1
> MIRROR_IMAGE_TAG: 0.84.0

File: .env [hunk 1]
```diff
@@ -4,8 +4,8 @@ NETWORK_NODE_IMAGE_PREFIX=gcr.io/hedera-registry/
 NETWORK_NODE_IMAGE_NAME=main-network-node
 
 #### Image Tags/Hashes ####
-NETWORK_NODE_IMAGE_TAG=0.38.6
-HAVEGED_IMAGE_TAG=0.38.6
+NETWORK_NODE_IMAGE_TAG=0.40.0-alpha.1
+HAVEGED_IMAGE_TAG=0.40.0-alpha.1
```
Comment (Nana-EC) [inline .env:8]:
```suggestion
HAVEGED_IMAGE_TAG=0.40.1
```

**Comment by Ivo-Yankov** [comment]
Changed the condition of applying the different `config.txt` formats. Now the applied network node version is used instead of the selected `--network` option. This way even if a specific version is passed as an inline env var the correct config.txt will be applied.

File: .env [hunk 2]
```diff
@@ -4,8 +4,8 @@ NETWORK_NODE_IMAGE_PREFIX=gcr.io/hedera-registry/
 NETWORK_NODE_IMAGE_NAME=main-network-node
 
 #### Image Tags/Hashes ####
-NETWORK_NODE_IMAGE_TAG=0.38.6
-HAVEGED_IMAGE_TAG=0.38.6
+NETWORK_NODE_IMAGE_TAG=0.40.0-alpha.1
```
[the remark this answers is not in the document]
Comment (Ivo-Yankov) [reply .env]: Done

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**